ribotTeamtvOS/RibotTeamViewController.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  RibotTeamViewController.swift
+//  ribotTeam
+//
+//  Created by Manuel Marcos Regalado on 26/01/2016.
+//  Copyright © 2016 Manuel Marcos Regalado. All rights reserved.
+//
+
+import UIKit
+
+class RibotTeamViewController: SharedRibotTeamViewController  {
+    func collectionView(collectionView: UICollectionView, didUpdateFocusInContext context: UICollectionViewFocusUpdateContext, withAnimationCoordinator coordinator: UIFocusAnimationCoordinator) {
+        if let indexPath = context.previouslyFocusedIndexPath,
+            cell = collectionView.cellForItemAtIndexPath(indexPath) as? SharedRibotTeamViewCell {
+                coordinator.addCoordinatedAnimations({
+                    cell.ribotNameLabel.alpha = 0.0
+                    }, completion: nil)
+        }
+        
+        if let indexPath = context.nextFocusedIndexPath,
+            cell = collectionView.cellForItemAtIndexPath(indexPath) as? SharedRibotTeamViewCell {
+                coordinator.addCoordinatedAnimations({
+                    cell.ribotNameLabel.alpha = 1.0
+                    }, completion: nil)
```

Comment:
Yep, if the return type on a block is void you can omit it so it'd default to void... I believe I understood that right... 
Reference: https://thatthinginswift.com/completion-handlers/